src/GenAPI/Microsoft.DotNet.GenAPI.Task/build/Microsoft.DotNet.GenAPI.Task.targets

```
@@ -47,6 +47,7 @@
       OutputPath="$(GenAPITargetPath)"
       HeaderFile="$(GenAPIHeaderFile)"
       ExceptionMessage="$(GenAPIExceptionMessage)"
+      ExcludeApiFiles="$(GenAPIExcludeApiFiles)"
```

MSBuild items are singular:

```suggestion
      ExcludeApiFiles="$(GenAPIExcludeApiFile)"
```

---

Renamed to `$(GenAPIExcludeApiList)`